pages/faq/llm-not-using-my-docs.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Why does the LLM not use my documents"
+title: "Why does the LLM is not using my documents"
```

Comment:
The correct language here is
`Why is the LLM not using my documents`. The original text was okay as well and did not need to be changed

Reply:
Ahh okay! I reverted the change back to the original (: